apps/condo/domains/ticket/schema/ExportTicketsService.js

```diff
@@ -59,103 +23,29 @@ const ExportTicketsService = new GQLCustomSchema('ExportTicketsService', {
-                        isEmergency: ticket.isEmergency ? 'X' : '',
-                        isWarranty: ticket.isWarranty ? 'X' : '',
-                        isPaid: ticket.isPaid ? 'X' : '',
-                        classifier: ticket.classifier || '',
-                        place: ticket.placeClassifier || '',
-                        category: ticket.categoryClassifier || '',
-                        description: ticket.problemClassifier || '',
-                        createdAt: formatDate(ticket.createdAt),
-                        updatedAt: formatDate(ticket.updatedAt),
-                        inworkAt: ticket.startedAt ? formatDate(ticket.startedAt) : '',
-                        completedAt: ticket.completedAt ? formatDate(ticket.completedAt) : '',
-                        status: indexedStatuses[ticket.status],
-                        operator: ticket.operator || ticket.createdBy || '',
-                        executor: ticket.executor || '',
-                        assignee: ticket.assignee || '',
-                        organizationComments: organizationCommentsToRender.join(TICKET_COMMENTS_SEPARATOR),
-                        residentComments: residentCommentsToRender.join(TICKET_COMMENTS_SEPARATOR),
-                        source: ticket.source || '',
-                        deadline: ticket.deadline ? formatDate(ticket.deadline) : '',
-                        reviewValue: ticket.reviewValue ? reviewValueText[ticket.reviewValue] : '',
-                        reviewComment: ticket.reviewComment || '',
-                        statusReopenedCounter: ticket.statusReopenedCounter || '',
-                    }
-                })
-
-                const linkToFile = await createExportFile({
-                    fileName: `tickets_${dayjs().format('DD_MM')}.xlsx`,
-                    templatePath: './domains/ticket/templates/TicketsExportTemplate.xlsx',
-                    replaces: {
-                        tickets: excelRows,
-                        i18n: {
-                            ...getHeadersTranslations(EXPORT_TYPE_TICKETS, locale),
-                            sheetName: i18n('excelExport.sheetNames.tickets', { locale }),
-
-                            // These statuses are needed to set cell color using conditional formatting by status name.
-                            statusNames: ticketStatusesTranslations(locale),
-                        },
-                    },
-                    meta: {
-                        listkey: 'Ticket',
-                        id: allTickets[0].id,
-                    },
+                const task = await startExportTicketsTask(context, {
+                    dv,
+                    sender,
+                    format: EXCEL,
+                    where,
+                    sortBy,
+                    locale,
+                    timeZone,
                 })
 
-                return { status: 'ok', linkToFile }
+                return { task }
```

Comment:
probably, it's better to return the `ExportTicketTask` as output?

Reply:
Or to use something like `SendMessageToSupportOutput`

Reply:
Yes, it was an old code